app/ledger.js

```diff
@@ -414,7 +414,21 @@ var enable = (onoff) => {
     } catch (ex) {
       console.log('synopsisPath parse error: ' + ex.toString())
     }
-    if (process.env.NODE_ENV === 'test') synopsis.options.minDuration = 0
```

Comment:
Minor: To help one find the default value of minDuration, it might be helpful to link to [ledger-publisher](https://github.com/brave/ledger-publisher/blob/3e915ca958455584dd93476fadc0fb49893dc445/index.js#L167) or the respective [readme](https://github.com/brave/ledger-publisher/blob/a408633f0d6f8a4f6cefc298b358e4acb43d7147/README.md#page-visits)